front/app/containers/Authentication/useSteps/index.ts

```diff
@@ -58,7 +58,17 @@ export default function useSteps() {
     return authenticationDataRef.current;
   }, []);
 
-  const [currentStep, setCurrentStep] = useState<Step>('closed');
+  const [currentStep, _setCurrentStep] = useState<Step>('closed');
```

Comment:
Small change: instead of always invalidating all cache related to action descriptor when the state transition is called `CLOSE`, we now do this when the next step is `closed`. Almost the same, with the exception that this now also covers cases where other types of transitions set the next step as `closed`.